build.properties

```diff
@@ -4,12 +4,12 @@ mod_id=RTG
 mod_name=Realistic Terrain Generation
 mod_desc=Adds a new world type which generates realistic terrain for Overworld biomes.
 mod_version=2.0.0-alpha.6
-mc_version=1.9.4
+mc_version=1.10.2
 mod_url=https://github.com/Team-RTG/Realistic-Terrain-Generation
 mod_author="Team RTG"
 mod_creds=Forked from RWG by ted80. OpenSimplex Noise by KdotJPG.
 mod_logo=assets/rtg/logo.png
-mcf_version=12.17.0.1976
+mcf_version=12.18.1.2011
```

Comment:
You forgot to update the mappings. stable_26 is for 1.9.4. Change it to snapshot_nodoc_20160721